playground/mocks/data/idp/idx/error-feature-not-enabled.json

```diff
@@ -0,0 +1,4 @@
+{
+  "error": "access_denied",
+  "error_description": "The requested feature is not enabled in this environment."
```

Comment:
are we okay to expose this to enduser ? should we more specific or clear in our messaging ?

Reply:
this is mock, but I copied from the actual response I received from the server